style.scss

```diff
@@ -20,6 +20,6 @@
 
 .vl-modal {
     .vl-modal-dialog {
-        overflow: visible;
+        overflow: auto;
```

Comment:
Ik ga even controleren of dit geen problemen introduceert voor #61. Ik zal dan ook een demo toevoegen hiervoor.

Reply:
Mijn vermoeden werd bevestigd. #61 is broken door deze wijzigingen. Ik heb een demo toegevoegd ter illustratie. Hiervoor kan best een e2e test toegevoegd worden om in de toekomst te vermijden dat deze bug opnieuw geïntroduceerd wordt.